docs/resources/User.md

```diff
@@ -78,6 +78,7 @@ There are other rules and restrictions not shared here for the sake of spam and
 | 1 << 16 | Verified Bot                 |
 | 1 << 17 | Early Verified Bot Developer |
 | 1 << 18 | Discord Certified Moderator  |
+| 1 << 19 | Bot HTTP Interactions        |
```

Comment:
What does `Bot HTTP Interactions` means? Even as someone who is used to the Discord API, I don't know what this means.

Reply:
A bot that doesn't connect to the gateway

Reply:
Ok, I think I understood

A description like `Bot using only [outgoing webook interaction](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/receiving-an-interaction) and always shown online` (or something like that) is better in my mind :)

Reply:
yeah definitly

Reply:
Gateway Status?

Reply:
bot status I would say

Reply:
this table is pretty weird because instead of having an actual name field it has a "description" field which is generally pretty close to the name, and libraries usually just normalize the description for a name

Reply:
Yeah, having both `name` and `description` could be useful